src/components/UI/Controls/ConfirmationPrompt/index.tsx

```diff
@@ -10,6 +10,7 @@ interface IConfirmationPromptProps
   cancelButton?: React.ReactNode;
   onConfirm?: () => void;
   onCancel?: () => void;
+  contentProps?: React.ComponentPropsWithoutRef<typeof Box>;
```

Comment:
Added a new prop to pass in contentProps to style the inner Box (i.e. the padding and background colour)